src/more-infos/more-info-group.js

```diff
@@ -36,4 +39,40 @@ export default new Polymer({
       ],
     },
   },
+
+  observers: [
+    'statesChanged(stateObj, states)',
+  ],
+
+  statesChanged(stateObj, states) {
+    let groupDomainStateObj = false;
+
+    if (states && states.length > 0) {
+      const baseStateObj = maxBy(states, (s) => Object.keys(s.attributes).length);
```

Comment:
Hmm now that I think about it more I agree. It will be too unpredictable.

Picking the state with the most attributes was intended to expose the entity with the most capabilities. For example I have multiple lights in a group but only some of them support color. I want to color controls for the group even though some of the lights don't support color (they will just ignore the setting).

Are the states always in the order they are defined in the configuration.yaml? If so then I think picking the first state is no problem. Users can just pick whatever entity they want the controls for and put it first in their groups.